packages/docs/src/fetch-polyfill.js

```diff
@@ -0,0 +1,45 @@
+// TODO: Fix the tests. Seems like it will only work properly once moved to Vitest or away from CRA.
```

Comment:
To prevent confusion, I'd suggest to rename this to `jest-polyfills.js` instead. This module doesn't really polyfill anything. It forces Jest to use Node.js globals that Jest purposefully force-removes from the test environment.